docs_src/security/SeedingServiceSecrets.md

```diff
@@ -0,0 +1,88 @@
+# Seeding Service Secrets
+
+!!! edgey "EdgeX 2.1"
+    New for EdgeX 2.1 is the ability to seed service specific secrets during the service's start-up. 
+
+All EdgeX services now have the capability to specify a JSON file that contains the service's secrets which are seeded into the service's `SecretStore` during service start-up. This allows the secrets to be present in the service's `SecretStore` when the service needs to use them.
+
+The new `SecretsFile` setting on the `SecretStore` configuration allows the service to specify the fully-qualified path to the location of the service's secrets file. Normally this setting is left blank when a service has no secrets to be seeded.
+
+!!! example "Example setting SecretsFilePath in TOML"
+
+    ```toml
+    [SecretStore]
+    Type = "vault"
+    ...
+    SecretsFile = "/tmp/my-service/secrets.json"
+    ...
+    ```
+
+This setting can also be overriding with the `SECRETSTORE_SECRETSFILE` environment variable in the docker-compose file and then volume mounted in to the service's container.
+
+!!! example "Example setting SecretsFilePath via environment override"
+    ```yaml
+    environment:
+      SECRETSTORE_SECRETSFILEPATH: "/tmp/my-service/secrets.json"

```

Comment:
Shouldn't this be `SECRETSTORE_SECRETSFILE`?

Reply:
Yes, I changed it and missed changing this one ;-)

Reply:
fixed